ganga/GangaCore/GPIDev/Adapters/IVirtualization.py

```diff
@@ -0,0 +1,35 @@
+##########################################################################
+# Ganga Project. http://cern.ch/ganga
+#
+##########################################################################
+from GangaCore.GPIDev.Schema import Schema, Version, SimpleItem, GangaFileItem
+from GangaCore.GPIDev.Base import GangaObject
+from GangaCore.GPIDev.Lib.File.File import File
+
+class IVirtualization(GangaObject):
+
+    """
+    Contains the interface for all virtualization classes, all virtualization classes should inherit from this object.
+    """
+
+    def __init__(self, image):
+        super(IVirtualization, self).__init__()
+        self.image = image
+
+    _schema = Schema(Version(1, 0), {
+        'image': SimpleItem(defvalue="", typelist=[str], doc='Link to the container image')
```

Comment:
Why did we remove the `GangaFile` from the typelist?

Reply:
I understood this when looking at the Singularity vs Docker concrete implementations. We can't give a file to Docker but we can have a standalone singularity image.